receiver_custom.html

```diff
@@ -403,7 +406,24 @@ <h3 id= "error_description" class ="cast_text"></h3>
       // prepare the player load script
       var playerJs = document.createElement("script");
       playerJs.type = "text/javascript";
-      playerJs.src = OOYALA_PLAYER_URL + "?" + (debug? ("debug=true&" + v3Version) : v3Version);
+      playerJs.src = OOYALA_PLAYER_URL;
+      if (debug || v3Version) {
+        if (!playerJs.src.match(/\?$/)) {
+          playerJs.src += "?";
+        } else if (playerJs.src.match(/\?[^=]+=/)) {
+          // URL contains ?, therefore it should contain name=value pair and we need to append "&"
+          playerJs.src += "&";
```

Comment:
I am assuming that it is a well-formed URL, so that name=value pairs follow quesiton mark.  You think that we should also consider cases when URL has format "url?name" ?